frontend/components/dashboard/DashboardTransactionsRow.js

```diff
@@ -1,8 +1,54 @@
+import mustache from "mustache";
 import { Row, Col } from "react-bootstrap";
 
 import Timer from "../utils/Timer";
 import GetTransactionIcon from "../utils/GetTransactionIcon";
 
+const prettifyTransaction = (kind, args) => {
+  if (kind === "AddKey") {
+    try {
+      args = JSON.parse(args);
```

Comment:
I think, we should resolve args once in the DataProvider instead of doing it every single time.

In fact, I consider moving this parsing to the backend side, but that will require adding an additional option to the `.select` RPC call as sequelize does not know which columns of the raw SELECT are of JSON type.

Reply:
Yes. It makes sense to do it in DataProvider once. I'll shift it over there.